StravaSwift/OAuthToken.swift

```diff
@@ -17,16 +17,32 @@ public struct OAuthToken: Strava {
     /** The access token **/
     public let accessToken: String?
     
+    /** The refresh token **/
+    public let refreshToken: String?
+    
+    /** Expiry for the token in seconds since the epoch **/
+    public let expiresAt : Int?
+    
     /** The athlete **/
     public let athlete: Athlete?
 
     /**
-     Initializer
+     Initializers
      
      - Parameter json: A SwiftyJSON object
      **/
     public init(_ json: JSON) {
         accessToken = json["access_token"].string
+        refreshToken = json["refresh_token"].string
+        expiresAt = json["expires_at"].int
         athlete = Athlete(json["athlete"])
     }
-}
\ No newline at end of file
+    
+    public init(access: String?, refresh: String?, expiry: Int?) {
```

Comment:
Should we also pass in `athlete` as an optional parameter with a nil default value?
`, athlete : Athlete? = nil`

Reply:
That sounds sensible...